edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/KubeUtils.cs

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes
+{
+    using System;
+    using System.Linq;
+    using System.Text;
+    using CoreConstants = Microsoft.Azure.Devices.Edge.Agent.Core.Constants;
+
+    public static class KubeUtils
+    {
+        const string Alphabet = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+        const string AllowedCharsLabelValues = "-._";
+        const string AllowedCharsDns = "-";
+        const string AllowedCharsGeneric = "-.";
+        const int MaxK8SValueLength = 253;
+        const int MaxDnsNameLength = 63;
+        const int MaxLabelValueLength = 63;
+
+        static bool IsAlpha(char ch) => Alphabet.IndexOf(ch) != -1;
```

Comment:
What do you think about using hash set as a collection for alphabet? It will give us constant time of checking symbols against alphabet as opposite to linear scan in array?

Reply:
Good idea!